editor/actions.js

```diff
@@ -386,3 +386,74 @@ export const createSuccessNotice = partial( createNotice, 'success' );
 export const createInfoNotice = partial( createNotice, 'info' );
 export const createErrorNotice = partial( createNotice, 'error' );
 export const createWarningNotice = partial( createNotice, 'warning' );
+
+/**
+ * Returns an action object used to fetch a single reusable block or all
+ * reusable blocks from the REST API into the store.
+ *
+ * @param {string} id If given, only a single reusable block with this ID will be fetched
+ * @return {Object}   Action object
+ */
+export function fetchReusableBlocks( id = null ) {
```

Comment:
Minor: do we really need the default value?

Reply:
I suppose not! Removed in 7762cb73a84b08f60938bc004cfd1fa20bccf36a.